lab/lab2/js/part3-application.js

```diff
@@ -23,11 +23,14 @@
   Remember, this is open-ended. Open ended doesn't mean pointless: we're looking for
   filters that might actually be useful. Try to see what you can produce.
 ===================== */
-
+var phillyCrimeDataUrl = "https://raw.githubusercontent.com/CPLN692-MUSA611/datasets/master/json/philadelphia-crime-snippet.json";
 /* =====================
   Define a resetMap function to remove markers from the map and clear the array of markers
 ===================== */
-var resetMap = function() {
+var resetMap = function(myMarkers) {
```

Comment:
I can't get your `resetMap()` to work.

I think it's because `myMarkers` has `undefined` elements in, as per this screen shot:
![image](https://user-images.githubusercontent.com/8103418/52920984-64664d80-32e0-11e9-9b6e-475d16418fc5.png)

I'm not exactly sure why you have `undefined`s in there because your conditional logic looks good to filter those out. But alas, it has `undefined`.